VideoLocker/res/layout/fragment_handout.xml

```diff
@@ -1,27 +1,41 @@
 <?xml version="1.0" encoding="utf-8"?>
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:background="@color/grey_act_background"
-    android:orientation="vertical" >
 
-    <org.edx.mobile.view.custom.EdxWebView
-        android:id="@+id/webview"
-        android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:background="@color/grey_act_background" />
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+            xmlns:tools="http://schemas.android.com/tools"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:background="@color/edx_grayscale_neutral_white"
+            tools:context="org.edx.mobile.view.CourseHandoutActivity">
 
-    <TextView
-        android:id="@+id/downloadMessage"
-        style="@style/downloading_message"
-        android:visibility="gone"
-        android:layout_alignParentTop="true"/>
+    <RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
```

Comment:
If you want to ensure it's *at least* as tall as the scrollview, set fillViewport=true on the ScrollView